plugin-interface/src/main/scala/mesosphere/marathon/plugin/RunSpec.scala

```diff
@@ -114,53 +119,18 @@ trait PodSpec extends RunSpec {
     * The labels in that pod.
     */
   val labels: Map[String, String]
-
-  /**
-    * Pod volumes
-    */
-  val podVolumes: Seq[PodVolumeSpec]
 }
 
-/**
-  * Application volume definition
-  */
-trait AppVolumeSpec {
-
-  val containerPath: String
+trait VolumeSpec {
+  val name: Option[String]
 }
 
-/**
-  * Application volume with a secret
-  */
-trait AppSecretVolumeSpec extends AppVolumeSpec {
-
+trait SecretVolumeSpec extends VolumeSpec {
   val secret: String
```

Comment:
Nit: add field comments here and below (I know they're self-explanatory but we do anyway since it's plugin-interface)

Reply:
Done.